opentelemetry-example/src/main/scala/zio/telemetry/opentelemetry/example/BackendServer.scala

```diff
@@ -1,39 +1,50 @@
 package zio.telemetry.opentelemetry.example
 
-import org.http4s.server.{ Router, _ }
+import org.http4s.server.{Router, defaults}
 import org.http4s.server.blaze.BlazeServerBuilder
 import zio.clock.Clock
+import zio.config.getConfig
+import zio.config.typesafe.TypesafeConfig
+import zio.config.magnolia.DeriveConfigDescriptor.descriptor
 import zio.interop.catz._
 import zio.telemetry.opentelemetry.Tracing
-import zio.telemetry.opentelemetry.example.config.{ Config, Configuration }
-import zio.telemetry.opentelemetry.example.http.{ AppEnv, AppTask, Client, StatusService }
-import zio.{ ExitCode, Managed, ZIO, ZLayer }
+import zio.telemetry.opentelemetry.example.config.AppConfig
+import zio.telemetry.opentelemetry.example.http.{AppEnv, AppTask, Client, StatusService}
+import zio.{ExitCode, Managed, ZIO, ZLayer}
 import org.http4s.syntax.kleisli._
 import sttp.client.asynchttpclient.zio.AsyncHttpClientZioBackend
+import sttp.model.Uri
 
 object BackendServer extends zio.App {
+  implicit val sttpUriDescriptor: zio.config.magnolia.Descriptor[Uri] =
+    zio.config.magnolia.Descriptor[String].transformOrFailLeft(str =>
```

Comment:
I'd import it, e.g.:

```scala
import zio.config.magnolia.Descriptor
```

Same with zio.App.

Reply:
Using `Descriptor` directly is deprecated.

Reply:
What does that mean? You're using it anyhow, I'm suggesting you to import it, as illustrated [here](https://zio.github.io/zio-config/docs/automatic/automatic_index#is-that-the-only-way-for-custom-derivation-)

Reply:
https://github.com/zio/zio-config/blob/68e41920ad7c3f74af379ebc5dbd228f05ecc43c/magnolia/shared/src/main/scala/zio/config/magnolia/DeriveConfigDescriptor.scala#L661

Reply:
Please read what I wrote [here](https://github.com/mvelimir/zio-telemetry/pull/4#discussion_r680906406).

Reply:
I get it now.